### PR TITLE
@W-8727016@ Update issue templates

### PR DESCRIPTION
**Pull request by ranekere-sfdc**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (jfeingold35) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: Do we want "Actual behavior" to also be a section here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Scanner Version [e.g. 22]
+
+**Additional context**
```
Comment (jfeingold35) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: "Workaround": Have you found any ways to sidestep the problem?
"Urgency": How severe is this problem? "Business stopping" vs "Not business-stopping".

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+
```
Comment (jbartolotta-sfdc) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: There seems to be an extra line here

**Review by jbartolotta-sfdc — APPROVED** [review]
Looks good, one small comment.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
```
Comment (jfeingold35) [inline .github/ISSUE_TEMPLATE/feature_request.md:19]: "Workaround": Can you do something similar with existing capabilities?
"Urgency": How badly do you need this feature? "Nice to have" vs "Highly beneficial" vs "Can't live without it".

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ranekere-sfdc) [reply .github/ISSUE_TEMPLATE/bug_report.md:19]: **Describe the bug** section should address this right? That is where they say what the actual behavior is.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (ranekere-sfdc) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: Good idea. Will add this.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jfeingold35) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: Good point.

**Review by jfeingold35 — APPROVED** [review]

**Pull request merged**